resources/views/service/partials/masthead.blade.php

```diff
@@ -1,8 +1,18 @@
 <div class="header">
     <img src="{{ asset('assets/logo.png') }}">
     <h1>Rose Vouchers</h1>
+    @if (!Auth::guest())
     <ul>
-        <li>User: {{ $user_name }}</li>
-        <li>Centre: {{ $centre_name }}</li>
+        <li>User: {{ Auth::user()->name }}</li>
+        <li>Centre: {{ Auth::user()->centre->name }}</li>
```

Comment:
This will throw fatal trying to get prop of non-object - if User centre not found for some reason. I'd wrap in an if just in case. I know it's not expected... but :)

Reply:
agreed